examples/binding-patterns-in-match-statement/binding_patterns_in_match_statement.metatags

```diff
@@ -1,2 +1,2 @@
 description: This BBE demonstrates binding patterns in a `match` statement.
-keywords: ballerina, ballerina by example, bbe, spread operator, binding pattern
+keywords: ballerina, ballerina by example, bbe, spread operator, binding pattern, match
```

Comment:
The description should be changed.
Please check https://github.com/ballerina-platform/ballerina-distribution/pull/3910#discussion_r1065326880

Reply:
Addressed in e7e755142d12ac2d41edf1411c334d63f3fd0b6d